vocabularies/PDF.xml

```diff
@@ -77,7 +77,7 @@
         <Property Name="Padding" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
           <Annotation Term="Core.Description" String="Padding of document" />
           <Annotation Term="Core.LongDescription">
-            <String>Sets the padding (left, right, bottom, top) in PDF units. The value must be > 0</String>
+            <String>Sets the padding (left, right, bottom, top) in PDF units</String>
```

Comment:
Shouldn't the text rather be something like
```suggestion
            <String>Padding (left, right, bottom, top) is supported</String>
```

Reply:
Yes right. 

==> Changed the description.